stable/prometheus-mysql-exporter/values.yaml

```diff
@@ -33,13 +33,20 @@ tolerations: []
 
 affinity: {}
 
-annotations: {}
-#  prometheus.io/scrape: "true"
-#  prometheus.io/path: "/metrics"
-#  prometheus.io/port: "{{ .Values.service.internalPort }}"
+annotations:
+  # prometheus.io/scrape: "true"
+  # prometheus.io/path: "/metrics"
+  # prometheus.io/port: 9104
```

Comment:
wasn't it better with the dinamic value ?

Reply:
Imho this will not work. This is the output of: "helm template ." 

```

---
# Source: prometheus-mysql-exporter/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: RELEASE-NAME-prometheus-mysql-exporter
  labels:
    app: prometheus-mysql-exporter
    chart: prometheus-mysql-exporter-0.2.0
    release: RELEASE-NAME
    heritage: Tiller
spec:
  type: ClusterIP
  ports:
    - port: 9104
      targetPort: 9104
      protocol: TCP
      name: mysql-exporter
  selector:
    app: prometheus-mysql-exporter
    release: RELEASE-NAME

---
# Source: prometheus-mysql-exporter/templates/deployment.yaml
apiVersion: apps/v1beta2
kind: Deployment
metadata:
  name: RELEASE-NAME-prometheus-mysql-exporter
  labels:
    app: prometheus-mysql-exporter
    chart: prometheus-mysql-exporter-0.2.0
    release: RELEASE-NAME
    heritage: Tiller
spec:
  replicas: 1
  selector:
    matchLabels:
      app: prometheus-mysql-exporter
      release: RELEASE-NAME
  template:
    metadata:
      labels:
        app: prometheus-mysql-exporter
        release: RELEASE-NAME
      annotations:
        prometheus.io/path: /metrics
        prometheus.io/port: '{{ .Values.service.internalPort }}'
        prometheus.io/scrape: "true"
        
    spec:
      containers:
        - name: prometheus-mysql-exporter
          image: "prom/mysqld-exporter:v0.11.0"
          imagePullPolicy: IfNotPresent
          env:
            - name: DATA_SOURCE_NAME
              value: "exporter:password@(localhost:3306)/"
          ports:
            - containerPort: 9104
          livenessProbe:
            httpGet:
              path: /
              port: 9104
          readinessProbe:
            httpGet:
              path: /
              port: 9104
          resources:
            {}
            

---
# Source: prometheus-mysql-exporter/templates/secret.yaml

```